.github/workflows/build-push-images-latest.yaml

```diff
@@ -5,14 +5,111 @@ on:
     branches:
       - develop
 jobs:
+  lint:
```

Comment:
This file is very similar with `.github/workflows/build-push-images-pr.yaml`.

The main difference is just it always push the docker image in `docker-builder` job.

Reply:
Maybe we can merge these files.